Simple-Projects/Simple_Projects/Car/Point.cpp

```diff
@@ -1,11 +1,11 @@
+#include <cmath>
```

Comment:
These Point.* files are duplicate ;(
I copied the changes into them so they're the same